client/src/entrypoints/contrib/typed_table_block/typed_table_block.js

```diff
@@ -221,7 +222,8 @@ export class TypedTableBlock {
     const prependColumnButton = $(`<button type="button"
       class="button button-secondary button-small button--icon text-replace prepend-column"
       aria-label="${h(this.blockDef.meta.strings.INSERT_COLUMN)}"
-      title="${h(this.blockDef.meta.strings.INSERT_COLUMN)}">
+      title="${h(this.blockDef.meta.strings.INSERT_COLUMN)}"
+      aria-expanded="false">
         <svg class="icon icon-plus icon" aria-hidden="true" focusable="false"><use href="#icon-plus"></use></svg>
```

Comment:
There's also an `appendColumnButton` (`data-append-column` in the initial HTML fragment near the top of the script) that needs the same change. Perhaps it'll be clearer to set this attribute alongside the point where we attach the onclick handler (`this.appendColumnButton.on('click', ...)`, `this.prependColumnButton.on('click', ...)`) rather than putting it in the initial HTML fragments?